infomaniak-build-tools/linux/package-ci-amd64.sh

```diff
@@ -56,16 +56,22 @@ $HOME/desktop-setup/linuxdeploy-x86_64.AppImage --appdir $APP_DIR -e $APP_DIR/us
 
 VERSION=$(grep "KDRIVE_VERSION_FULL" "$BASE_DIR/build-linux/build/version.h" | awk '{print $3}')
 APP_NAME=kDrive-${VERSION}-amd64.AppImage
-mv kDrive*.AppImage $APP_NAME
+
+mv kDrive*.AppImage ../$APP_NAME
+cd ..
 
 if [ -z ${KDRIVE_TOKEN+x} ]; then
 	echo "No kDrive token found, AppImage will not be uploaded."
 else
-	APP_SIZE=$(ls -l $APP_NAME | awk '{print $5}')
+	FILES=($APP_NAME "kDrive-amd64.dbg" "kDrive_client-amd64.dbg")
```

Comment:
The computation of the size of the app executable and the for-loop below could done in a function that takes os-specific arguments for Linux and MacOSX (I don't if it's possible for Windows).

Reply:
Done for MacOSX/Linux
Windows unfortunately uses its own powershell function instead of curl (it is only an alias) so it cannot be integrated with this change